.editorconfig

```diff
@@ -0,0 +1,13 @@
+# top-most EditorConfig file
+root = true
+
+[*]
+# Unix-style newlines with a newline ending every file
+end_of_line = lf
```

Comment:
what does this do?

Reply:
It resolves whether to use Unix standard (lf), Mac OS 9 standard (cr), or Windows standard (cr+lf).

Reply:
You mean it forces to use `lf` irrespective of the OS?
@kaapstorm

Reply:
Yes. `.editorconfig` [sets the configuration](https://editorconfig.org/) for a bunch of different editors.